.github/workflows/docker.yaml

```diff
@@ -24,10 +27,20 @@ jobs:
       - uses: actions/checkout@v1
         with:
           fetch-depth: 1
-      - name: build
+      - name: build nightly image
+        if: github.event_name == 'schedule'
         env:
           IMAGE_NAME: ${{ secrets.DOCKER_USERNAME }}/nebula-${{ matrix.service }}:nightly
         run: |
           docker build -t ${IMAGE_NAME} -f docker/Dockerfile.${{ matrix.service }} .
           docker push ${IMAGE_NAME}
         shell: bash
+      - name: build release image
+        if: github.event_name == 'release'
+        env:
+          IMAGE_NAME: ${{ secrets.DOCKER_USERNAME }}/nebula-${{ matrix.service }}
+        run: |
+          TAG=$(git describe --exact-match --abbrev=0 --tags)
```

Comment:
Will the `TAG` empty?

Reply:
No. Because this job will be triggered by release event which will create a new tag on the current commit.

Reply:
No. Because this job will be triggered by release event which will create a new tag on the current commit.